### PR TITLE
NEXTEUROPA-10128 Design of language selection page in EC_Resp

### PR DESCRIPTION
**Pull request by yhuard**

### CODE REVIEW

**Comment by voidtek** [comment]
:+1:

**Comment by netlooker** [comment]
👍

**Comment by netlooker** [comment]
👍

**Comment by jfhovinne** [comment]
:+1:

File: profiles/common/themes/ec_resp/template.php [hunk 1]
```diff
@@ -1912,3 +1912,41 @@ function ec_resp_preprocess_comment(&$variables) {
...
+  $content .= _ec_resp_nexteuropa_multilingual_language_list_column($first_half, $variables['path'], $options);
+  $content .= _ec_resp_nexteuropa_multilingual_language_list_column($second_half, $variables['path'], $options);
+
+  $content .= '</div>';
+
+  return $content;
+}
+
+/**
+ * Helper function to display splash page language list column.
```
Comment (tatarbj) [inline profiles/common/themes/ec_resp/template.php]: missing @param and @return annotations.

**Comment by tatarbj** [comment]
:-1:

**Comment by tatarbj** [comment]
:-1:

**Comment by yhuard** [comment]
@tatarbj can you explain?

**Comment by tatarbj** [comment]
as it's written here: https://www.drupal.org/coding-standards/docs
the format is different, let me put the real solution here:
`/**
   * Helper function to display splash page language list column.
   *
   * @param array $languages 
   *    An associative array of languages to link to.
   * @param string $path
   *    The internal path being linked to.
   * @param array $options 
   *    An associative array of additional options.
   *
   * @return string 
   *    Formatted HTML column displaying the list of provided languages.
   */`

**Comment by tatarbj** [comment]
ahh, it's that ugly what i've put here, sorry for that.
The reason of my :-1: is the description of the annotations has to be in the next line and also between the annotation groups should be an empty line

**Comment by tatarbj** [comment]
http://pastebin.com/Ja1KUC02

**Comment by yhuard** [comment]
Thank you @tatarbj 

PS: the final line of your snippet misses a starting whitespace

**Comment by tatarbj** [comment]
beautiful code!

**Comment by tatarbj** [comment]
:+1:

**Comment by jfhovinne** [comment]
:+1:

**Comment by netlooker** [comment]
@yhuard check you pre push because this is second time in this PR that code standards checks are surpassed.

**Pull request merged**